### ISSUE TITLE
non-zero exit status 2 when running pyright

### ISSUE THREAD

**Issue by MarcSkovMadsen** [microsoft/pyright#3356]
I'm trying to learn how to use `pyright` on the [panel]() package to improve the way it works in VS Code. But I cannot run it. And the error message is not very helpful.

If I 

```bash
$pip install pyright
$pyright --verifytypes panel
```

I get a `non-zero exit status 2`.

```bash
$ pip install pyright
Looking in indexes: https://pypi.org/simple, https://masma:****@pkgs.dev.azure.com/dongenergy-p/_packaging/MarketTrading%40Local/pypi/simple/, https://masma:****@pkgs.dev.azure.com/dongenergy-p/_packaging/analytics-workspace/pypi/simple/
Collecting pyright
  Downloading pyright-1.1.238-py3-none-any.whl (15 kB)
Collecting nodeenv>=1.6.0
  Using cached nodeenv-1.6.0-py2.py3-none-any.whl (21 kB)
Installing collected packages: nodeenv, pyright
Successfully installed nodeenv-1.6.0 pyright-1.1.238
WARNING: You are using pip version 20.2.3; however, version 22.0.4 is available.
You should consider upgrading via the 'c:\repos\private\now\.venv\scripts\python.exe -m pip install --upgrade pip' command.     
(.venv) 
MASMA@PC81978 MINGW64 /c/repos/private/now
$ pyright --verifytypes panel
 * Environment already exists: C:\Users\masma\AppData\Local\Temp\pyright-python.MASMA\env
Traceback (most recent call last):
  File "C:\Program Files\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Program Files\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\repos\private\now\.venv\Scripts\pyright.exe\__main__.py", line 7, in <module>
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\cli.py", line 72, in entrypoint
    sys.exit(main(sys.argv[1:]))
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\cli.py", line 20, in main
    return run(*args, **kwargs).returncode
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\cli.py", line 36, in run
    npx = node.version('npx')
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\node.py", line 116, in version
    proc = run(target, '--version', stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\node.py", line 86, in run
    binary = _ensure_available(target)
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\node.py", line 34, in _ensure_available
    return Binary(path=_ensure_node_env(target), strategy=Strategy.NODEENV)
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\node.py", line 52, in _ensure_node_env
    _install_node_env()
  File "c:\repos\private\now\.venv\lib\site-packages\pyright\node.py", line 79, in _install_node_env
    subprocess.run(args, check=True)
  File "C:\Program Files\Python39\lib\subprocess.py", line 528, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['c:\\repos\\private\\now\\.venv\\scripts\\python.exe', '-m', 'nodeenv', 'C:\\Users\\masma\\AppData\\Local\\Temp\\pyright-python.MASMA\\env']' returned non-zero exit status 2.
```

My first suggestion would be to provide a better error message: Please tell me the cause and how to fix it.
Secondly it might also be an idea to improve the [installation instructions](https://github.com/microsoft/pyright#command-line) to tell that it is not enough to `pip install pyright` - you will need to also install node???? Its probably because I don't have node installed ??

## Additional Context

OS: Win 10 Enterprise
Terminal: Git Bash

**Comment by MarcSkovMadsen** [comment]
If I add `nodejs` to my `PATH`, I can run `pyright`.

**Comment by erictraut** [comment]
Yes, pyright does require node to be installed.

If you ran `pip install pyright`, then you are using [a community-maintained package](https://github.com/RobertCraigie/pyright-python) to install pyright through pip. If you are running into problems using this installation method or have suggestions for improved error messages, please file a bug with that project.
